package-manager-configs/nx-staging/deb/sonatype-community.list

```diff
@@ -1,2 +1,2 @@
-deb [arch=all] https://nx-staging.sonatype.com/repository/community-apt-hosted/ bionic main
-#deb https://nx-staging.sonatype.com/repository/community-apt-hosted/ bionic main
+deb [arch=all] https://repo.sonatype.com/repository/community-apt-hosted-staging/ bionic main
```

Comment:
I don't think these links should change because they refer to the old `nx-staging`. Similarly, the repo name (`community-hosted`) is the same on the staging server as the production server.

Or is a change a-foot to the nx-staging repo names? If so, there's lot's of other changes that will need to happen (e.g. CI config, etc).